src/Network/Flink/Internal/Stateful.hs

```diff
@@ -71,6 +86,8 @@ data Env = Env
   }
   deriving (Show)
 
+newtype ClToken = ClToken UUID
```

Comment:
Might be a good idea to make this serializable with either Protobuf or JSON. I expect users will be storing the token in their state to potentially cancel later.

Reply:
If someone wants to serialize a cancellation token, he can extract the underlying UUID and pass it on